cli/plasmo/src/type.ts

```diff
@@ -3,6 +3,42 @@ import type { ManifestContentScript } from "@plasmo/constants/manifest/content-s
 
 export type PlasmoContentScript = Omit<Partial<ManifestContentScript>, "js">
 
-export type PlasmoGetMountPoint = () => Promise<HTMLElement> | HTMLElement
+type GetT<T = HTMLElement> = () => T | Promise<T>
```

Comment:
Haven't tested but `T extends HTMLElement = HTMLElement` might be more flexible here?

Reply:
We want default type and not a base-minimum type. The `extends` actually enforce all T to be subclass of HTMLElement, which would be less flexible (GetT<string> would be invalid)